### DISCUSSION TITLE
Dompdf Failed to Render HTML output

### DISCUSSION THREAD

**Discussion by junaidimunto** [dompdf/dompdf#2707] [General]
How to solve this dompdf failed to render in my hosting, but it displayed normal pdf output view in localhost?
![dompdferrorrender](https://user-images.githubusercontent.com/72388586/147173698-c2571c5c-7f22-4924-ac13-61626a395beb.png)

**Comment by bsweeney** [comment]
The PDF _was_ rendered. You're looking at it, or at least the source of it. Something modified the HTTP response headers so that the browser didn't know that you were sending a PDF. To provide guidance we'd have to know more about the system and how you're generating the PDF.
